### PR TITLE
lint: upgrade to slim-v4

### PR DESCRIPTION
**Pull request by gwarf**
Slim version do not include rust, dotenv, armttk and pwsh: https://github.com/github/super-linter#slim-image

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ jobs:
 
       # Runs the Super-Linter action
       - name: Run Super-Linter on new changes
-        uses: github/super-linter@v3
+        uses: github/super-linter@slim-v4
```
Comment (enolfc) [inline .github/workflows/lint.yml]:
```suggestion
        uses: ghcr.io://github/super-linter@slim-v4
```

**Review by enolfc — CHANGES_REQUESTED** [review]

File: .github/workflows/lint.yml (hunk 1 above)
Comment (gwarf) [reply .github/workflows/lint.yml]: Oh! Good catch, thanks!

**Comment by gwarf** [comment]
So it still fails :/

**Comment by gwarf** [comment]
Saw this @enolfc : https://github.com/github/super-linter/issues/1596

File: .github/workflows/lint.yml [hunk 2]
```diff
@@ -18,7 +18,7 @@ jobs:
 
       # Runs the Super-Linter action
       - name: Run Super-Linter on new changes
-        uses: github/super-linter@v3
+        uses: ghcr.io://github/super-linter@slim-v4
```
Comment (enolfc) [inline .github/workflows/lint.yml:21]:
```suggestion
        uses: ghcr.io/github/super-linter@slim-v4
```

**Review by enolfc — CHANGES_REQUESTED** [review]
Nevermind...

**Review by enolfc — APPROVED** [review]
Finally works! Cool.

**Pull request merged**